index.js

```diff
@@ -93,10 +101,19 @@ async function main() {
                 await exec.exec('git', ['checkout', 'HEAD', '-B', branch]);
 
                 await exec.exec('git', ['commit', '-am', git_commit_message]);
+                
+                
                 const url = addToken(pr.head.repo.clone_url, token);
                 await exec.exec('git', ['push', url, 'HEAD']);
+
+                const sha = await getCommitHash('HEAD');
+                core.setOutput('committed', sha);
             });
+        } else {
+            core.setOutput('committed', '');
         }
+    } else {
+        core.setOutput('committed', '');
     }
```

Comment:
You can remove those else blocks and reduce the 3 core.submit lines to a single one

```suggestion
    }
    core.setOutput('committed', sha);
```

Reply:
You would have to set `let sha = ''` earlier before line 84